ui/package.json

```diff
@@ -34,7 +36,8 @@
     "ganache-cli": "^6.12.0",
     "lerna": "^3.22.1",
     "truffle": "^5.1.48",
-    "vercel": "^21.0.1"
+    "vercel": "^21.0.1",
+    "wait-on": "^5.2.1"
```

Comment:
what's this

Reply:
little cli util that will wait for a url or websocket or file to become available before returning. Used in the `yarn stack` script